index.html

```diff
@@ -1,11 +1,757 @@
+            with a mixture of jubilation and caution.
+          </p>
+          <p>
+            Max Tegmark, a cosmologist at M.I.T., wrote in an email, “I think
+            that if this stays true, it will go down as one of the greatest
+            discoveries in the history of science.”
+          </p>
+          <p>
+            John E. Carlstrom of the University of Chicago, Dr. Kovac’s mentor
+            and head of a competing project called the South Pole Telescope,
+            pronounced himself deeply impressed. “I think the results are
+            beautiful and very convincing,” he said.
+          </p>
+          <p>
+            Paul J. Steinhardt of Princeton, author of a competitor to inflation
+            that posits the clash of a pair of universes as the cause of
+            genesis, said that if true, the Bicep result would eliminate his
+            model, but he expressed reservations about inflation.
+          </p>
+          <p>
+            Lawrence M. Krauss of Arizona State and others also emphasized the
+            need for confirmation, noting that the new results exceeded earlier
+            estimates based on temperature maps of the cosmic background by the
+            European Space Agency’s Planck satellite and other assumptions about
+            the universe.
+          </p>
+          <p>
+            “So we will need to wait and see before we jump up and down,” Dr.
+            Krauss said.
+          </p>
+          <p>
+            Corroboration might not be long in coming. The Planck spacecraft
+            will report its own findings this year. At least a dozen other teams
+            are trying similar measurements from balloons, mountaintops and
+            space.
+          </p>
+          <h3>Spirals in the Sky</h3>
+          <p>
+            Gravity waves are the latest and deepest secret yet pried out of the
+            cosmic microwaves, which were discovered accidentally by Arno
+            Penzias and Robert Wilson at Bell Labs 50 years ago. They won the
+            Nobel Prize.
+          </p>
+          <p>
+            Dr. Kovac has spent his career trying to read the secrets of these
+            waves. He is one of four leaders of Bicep, which has operated a
+            series of increasingly sensitive radio telescopes at the South Pole,
+            where the thin, dry air creates ideal observing conditions. The
+            others are Clement Pryke of the University of Minnesota, Jamie Bock
+            of the California Institute of Technology and Dr. Kuo of Stanford.
+          </p>
+          <p>
+            “The South Pole is the closest you can get to space and still be on
+            the ground,” Dr. Kovac said. He has been there 23 times, he said,
+            wintering over in 1994. “I’ve been hooked ever since,” he said.
+          </p>
+          <img src="Images/telescope-img.jpg" alt="telescope image">
```

Comment:
`REQUIRED`

Images and videos should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image or video